src/cloud/project/project-conf-files_magento-app.md

```diff
@@ -11,8 +11,8 @@ The `.magento.app.yaml` has many default values, see [a sample `.magento.app.yam
 
 Use the `.magento.app.yaml` file to define the following configuration values:
 
-- [Properties]({{ site.baseurl }}/cloud/project/project-conf-files_magento-app_properties.html)—Define property values for application instance.
-- [Variables]({{ site.baseurl }}/cloud/project/project-conf-files_magento-app_variables.html)—Review environment variables required for the application instance.
-- [PHP Application]({{ site.baseurl }}/cloud/project/project-conf-files_magento-app_php-application.html)—Configure runtime PHP options.
-- [Workers]({{ site.baseurl }}/cloud/project/project-conf-files_magento-app_workers.html)—Create a worker instance and understand background tasks.
-- [Set Cache For Static Files]({{ site.baseurl }}/cloud/project/set-cache.html)—Set cache time-to-live (TTL) for your media and static files.
+-[Properties]({{ site.baseurl }}/cloud/project/project-conf-files_magento-app_properties.html)—Define property values for application instance.
+-[Variables]({{ site.baseurl }}/cloud/project/project-conf-files_magento-app_variables.html)—Review environment variables required for the application instance.
+-[PHP Application]({{ site.baseurl }}/cloud/project/project-conf-files_magento-app_php-application.html)—Configure runtime PHP options.
+-[Workers]({{ site.baseurl }}/cloud/project/project-conf-files_magento-app_workers.html)—Create a worker instance and understand background tasks.
+-[Set Cache For Static Files]({{ site.baseurl }}/cloud/project/set-cache.html)—Set cache time-to-live (TTL) for your media and static files.
```

Comment:
I can take the file names out the list - just wanted us to see what it looked like
The reason I took the spaces out was that lint tests were complaining about them!

I'll make the updates and we can see.

Reply:
@hguthrie - So apart from the proposed filename changes - how we looking for content?

BTW when I change the filenames, should I put this at the top of each and add the old file name :
redirect_from:
  - /cloud/deploy/configure-deploy.html

Reply:
I don't think redirects are necessary in this case. I think that redirect was created when the config file topics split out...but that was long ago.

Reply:
> I can take the file names out the list - just wanted us to see what it looked like
> The reason I took the spaces out was that lint tests were complaining about them!
> 
> I'll make the updates and we can see.

I think the order looks good. This will help to focus those topics for Workers and php.ini customizations.